p11mod/p11mod.go

```diff
@@ -57,7 +57,7 @@ func init() {
 type llSession struct {
 	session         p11.Session
 	slotID          uint
-	objects         []p11.Object
+	objects         []p11.Object //Used to keep track of mapping between object handles and objects, because object handles don't exist in the p11 API
```

Comment:
Can you put this comment on the preceding line (as a standalone line)?  It's long enough that putting it on the same line as the code makes it a bit hard to read on small screens.

Reply:
Okay!